code/modules/clothing/suits/armor_suits.dm

```diff
@@ -201,13 +201,13 @@
 
 /obj/item/clothing/suit/armor/vest/warden/alt
 	name = "warden's jacket"
-	desc = "A navy-blue armored jacket with blue shoulder designations and '/Warden/' stitched into one of the chest pockets."
+	desc = "A large navy-blue jacket fitted with Level II armored inserts. There are silver rank insignia on the shoulders."
 	icon_state = "warden_jacket_alt"
 
 //Captain
 /obj/item/clothing/suit/armor/vest/capcarapace
 	name = "captain's carapace"
-	desc = "An armored vest reinforced with ceramic plates and pauldrons to provide additional protection whilst still offering maximum mobility and flexibility. Issued only to the station's finest, although it does chafe your nipples."
+	desc = "A fancy blue & gold dragonscale armor vest fitted with Level III and IV armor panelling. Offers excellent protection against melee attacks, kinetic impacts, and laser discharges. "
```

Comment:
Finally, an armour vest that offers great protection AND doesn't chafe my nipples.